src/test/resources/TestNG.xml

```diff
@@ -7,6 +7,9 @@
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="Homework16"/>
+            <class name="Homework17"/>
+>>>>>>> Stashed changes
         </classes>
     </test>
 </suite>
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223643756-8e45af13-75c3-4752-87da-59e5f7ca21eb.png)